docker-compose.yml

```diff
@@ -33,29 +65,80 @@ services:
       - "8094"
     container_name: couchbase-sandbox-7.0.0
     profiles:
+      - server
       - local
       - mix-and-match
       - test
 
-  backend:
-    build: .
+  # ---------------------BACKENDS---------------------
```

Comment:
This section may be necessary, but does kinda show how complicated this is 😄

Reply:
This is probably only necessary for Python, the rest don't support Capella so there only need to be 2 backends